### PR TITLE
full bundle tests README

### PR DESCRIPTION
**Pull request by ColmBhandal**
Added readme to full bundle tests

### CODE REVIEW

File: tests-bundle/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+The tests are organized into directories based on the version of Kubeflow. Each version has its own suite of tests, although there may be similarities between versions. Minimally, each version's directory contains a Python file for deploying Kubeflow and one or more other files representing other tests.
+
+The test order is enforced to ensure that the deployment test runs and succeeds before further tests can run. This order is defined with the help of pytest markers, where the "deploy" tests are executed first.
+
+## How to Run the Tests as a GitHub Action
+
+To run the tests as a GitHub Action, follow these steps:
+
+1. Trigger the full bundle tests GitHub action (Tests) by clicking "Run workflow" in the repository.
```
Comment (beliaev-maksim) [inline tests-bundle/README.md:15]: can we add a link to the workflow?

File: tests-bundle/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+2. Provide the following inputs:
+
+   - **Folder**: Path from the content root to the directory for the version of Kubeflow you want to test.
```
Comment (beliaev-maksim) [inline tests-bundle/README.md:18]: need to add example

File: tests-bundle/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+## How to Run the Tests as a GitHub Action
+
+To run the tests as a GitHub Action, follow these steps:
+
+1. Trigger the full bundle tests GitHub action (Tests) by clicking "Run workflow" in the repository.
+2. Provide the following inputs:
+
+   - **Folder**: Path from the content root to the directory for the version of Kubeflow you want to test. 
+
+   - **TOX Posargs**: The deployment source for `juju deploy`. This will either be a published channel e.g. `--channel=1.7/stable` or the path from the content root of an unpublished `bundle.yaml` file within the repo.
```
Comment (beliaev-maksim) [inline tests-bundle/README.md:20]: there is no **TOX Posargs** field

File: tests-bundle/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+3. The GitHub action will start executing and perform the following steps:
```
Comment (beliaev-maksim) [inline tests-bundle/README.md:22]: we can mention that it runs on self hosted runners

File: tests-bundle/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+   - Set up the prerequisite installation and configuration for Kubeflow, including microk8s and Juju.
+   - Run the bundle tests, which are Python tests, to deploy Kubeflow and perform further tests on the deployed Kubeflow.
+   - Perform post-processing of the test results and gather information in case of failures.
+
+Please refer to the GitHub Action YAML file in the parent repository for more detailed information on the action setup and configuration.
+
+Note: This README assumes that you are familiar with GitHub Actions and have the necessary permissions to trigger and run actions in the repository.
```
Comment (beliaev-maksim) [inline tests-bundle/README.md:30]: maybe note should go first ? :D

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]

File: tests-bundle/README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+The test order is enforced to ensure that the deployment test runs and succeeds before further tests can run. This order is defined with the help of pytest markers, where the "deploy" tests are executed first.
+
+## How to Run the Tests as a GitHub Action
+
+These steps assume that you are familiar with GitHub Actions and have the necessary permissions to trigger and run actions in the repository.
+
+To run the tests as a GitHub Action, follow these steps:
+
+1. Trigger the full bundle tests [GitHub action](/.github/workflows/full-bundle-tests.yaml) by selecting `View Runs -> Run workflow`.
```
Comment (beliaev-maksim) [inline tests-bundle/README.md:17]: isn't it?

https://github.com/canonical/bundle-kubeflow/actions/workflows/full-bundle-tests.yaml

Comment (ColmBhandal) [reply tests-bundle/README.md]: I think I found a better way that solves both problems:

- Independent of the repo URL
- Links directly to the action (and not the YAML file)

Comment (ColmBhandal) [reply tests-bundle/README.md]: This seems to work OK in general. Although one potential issue is if GitHub changes their URL structure. I have examples of it working here: https://github.com/ColmBhandal/readmeabsolutelinks.

**Review by beliaev-maksim — APPROVED** [review]

**Pull request merged**